rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -766,3 +797,39 @@ def test_add_user_owned_asset_nft(globaldb):
     new_assets = {x[0] for x in result}
     assert new_assets - initial_assets == {A_DAI.identifier, A_PICKLE.identifier, A_CRV.identifier}
     assert all(not x.startswith(NFT_DIRECTIVE) for x in new_assets)
+
+
+def test_evm_token_deletion(globaldb):
+    token_data = EvmToken.initialize(
+        address=make_ethereum_address(),
```

Comment:
check that the information has been removed from all the tables and also test with a normal asset to verify that the logic is correct in all the cases